src/test/java/core/basesyntax/service/impl/CsvFileWriterTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvFileWriterTest {
+    private static final String REPORT_NAME = "src/test/resources/report.csv";
+    private static final FileWriter csvfileWriter = new CsvFileWriter();
+
+    @BeforeEach
+    void setUp() {
+        csvfileWriter.writeTextToFile(REPORT_NAME, "");
```

Comment:
do you really need to do it? 
what happens if your service will work incorrectly?